src/app/items/containers/permissions-edit-dialog-form/permissions-edit-form.component.ts

```diff
@@ -51,6 +67,8 @@ export class PermissionsEditFormComponent implements OnDestroy, OnChanges {
   };
 
   form = this.fb.group({
+    canEnterEnabled: [ false ],
+    canEnter: [ null ],
```

Comment:
Once we do this form as types supported, I would suggest to make those controls as optional.